### PR TITLE
[ronilaukkarinen__gulp-stylelint] Update stylelint dependency

### PR DESCRIPTION
**Pull request by martin-badin**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
- [ ] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test ronilaukkarinen__gulp-stylelint`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

**Comment by martin-badin** [comment]
@gabritto @peterblazejewicz Please review. I am the owner. Thank you

File: types/ronilaukkarinen__gulp-stylelint/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "private": true,
     "dependencies": {
-        "stylelint": "^14.0.1"
+        "stylelint": "*"
```
Comment (peterblazejewicz) [inline types/ronilaukkarinen__gulp-stylelint/package.json:4]: I believe this is not the best approach, exposes vector of attack.
Can this be more specific?
https://github.com/ronilaukkarinen/gulp-stylelint/blob/master/package.json#L35

Comment (martin-badin) [reply types/ronilaukkarinen__gulp-stylelint/package.json]: Thanks. The file is updated.

**Review by peterblazejewicz — APPROVED** [review]
LGTM!

**Comment by martin-badin** [comment]
@peterblazejewicz Thank you for the fast review.

**Comment by martin-badin** [comment]
Ready to merge

**Pull request merged**